frontend/src/App.tsx

```diff
@@ -107,20 +109,21 @@ function App() {
       onChangeTodo={onChangeTodo}
     ></TodoList>
   ) : !loading && todos.length === 0 && selectedFilter === Category.ALL ? (
-    <button onClick={onFocusInput} className="msg-box">
-      <p className="msg-text">
-        <b> No todos added yet!</b> Click here to add a new todo
-      </p>
-    </button>
+    <Alert onClick={onFocusInput} type="message">
+      <span>
+        <b>No todo added! </b>
+        click here to enter new todo
```

Comment:
It should be `Click` instead of `click`. After a `!`, we capitalize the next word.